assets/scss/components/citadel/forms/_forms.scss

```diff
@@ -137,6 +137,12 @@
 //    appear before the button.
 //
 // -----------------------------------------------------------------------------
+@mixin placeholder {
+    &::-webkit-input-placeholder {@content}
+    &::-moz-placeholder          {@content}
+    &:-moz-placeholder          {@content}
```

Comment:
I believe this rule is for Firefox 4-18 and we are not supporting it. Also could you please verify that this solution works in Edge?

Reply:
it's true. already removed. edge is supported
<img width="1658" alt="edge support" src="https://user-images.githubusercontent.com/67792608/103221522-8e9ce380-492b-11eb-9dd6-2c240a0b9e9b.png">

Reply:
👍